src/CftfBundle/Repository/LsItemRepository.php

```diff
@@ -166,4 +166,19 @@ public function createGradeSelectListQueryBuilder(): QueryBuilder
             ])
             ;
     }
+
+    /*
+     * Find all children items of selected item
+     */
+    public function allChilds(&$childIds, LsItem $lsItem)
+    {
+        $hasChildren = $lsItem->getChildren();
+        while (empty($hasChildren)) {
```

Comment:
this while loop should be removed since it is not required as the foreach() would just do not work with an empty list